rich-default/Dockerfile

```diff
@@ -233,37 +141,6 @@ RUN apt-get update && \
     apt-get clean
 ENV PATH=$PATH:/usr/lib/rstudio-server/bin
 
-# Installs the jupyter-rsession-proxy extension
-RUN conda install -yq -c conda-forge jupyter-rsession-proxy && \
-    conda clean -tipsy
-
-# Install pymol
-#RUN conda install -yq -c schrodinger pymol
-
-# Install ipymol
-# Installs from a git repo because pymol fetch fails when installing the pip package
-RUN pip install git+https://github.com/cxhernandez/ipymol.git@2a30d6ec1588434e6f0f72a1d572444f89ff535b
-
-RUN pip install pypdb
-RUN pip install biopandas
-
-USER $NB_USER
-
-
-# The following is an attempt to configure nb_conda_kernels automatically
-#WORKDIR $HOME
-#
-#USER root
-#
-#RUN echo "test" > $HOME/test.txt
-#
-## Create a folder of conda environments in the user's directory
-#RUN conda config && \
-#    > $HOME/.condarc && \
-#    echo "envs_dirs:" >> $HOME/.condarc && \
-#    echo "  -/home/jovyan/my-conda-envs/" >> $HOME/.condarc
```

Comment:
Why commented?

Reply:
I tried to automate installing nb_conda_kernels and this code does not work.